torch/distributed/pipelining/PipelineSchedule.py

```diff
@@ -652,14 +658,6 @@ def _step_microbatches(
 
 
 class ScheduleInterleaved1F1B(PipelineScheduleMulti):
-    """
-    The Interleaved 1F1B schedule.
-    Will perform one forward and one backward on the microbatches in steady
-    state and supports multiple stages per rank. When microbatches are ready for
-    multiple local stages, Interleaved 1F1B prioritizes the earlier microbatch
-    (also called "depth first").
-    """
-
```

Comment:
Replacement for this documentation?

Reply:
Accidentally removed it, I added it back in with additional comments